libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -651,6 +651,12 @@ bool AP_Arming::rc_arm_checks(AP_Arming::Method method)
         check_failed(ARMING_CHECK_PARAMETERS, true, "Mode channel and RC%d_OPTION conflict", rc().flight_mode_channel_number());
         check_passed = false;
     }
+
+    // The MAVLINK command arm does not check radio input.
```

Comment:
This doesn't look right; where's the check for which mode you're in?

Reply:
@peterbarker san.  Thanks for the comment.
When I access the AP_ARMING check from the copter ARMING check, I check the current flight mode.
The AP_ARMING check does not use the argument METHOD.
I added GUIDED to this METHOD.
I changed it to DO NOT CHECK RADIO INPUT if GUIDED.

AFTER
![Screenshot from 2022-06-18 10-47-48](https://user-images.githubusercontent.com/646194/174418156-6c2dd063-97b5-4da2-8696-68b2d52e83b2.png)